BinaryEncoding.md

```diff
@@ -170,52 +223,79 @@ The function section _declares_ the signatures of all functions in the
 module (their definitions appear in the [code section](#code-section)).
 
 | Field | Type | Description |
-| ----- |  ----- | ----- |
+| ----- | ---- | ----------- |
 | count | `varuint32` | count of signature indices to follow |
 | types | `varuint32*` | sequence of indices into the type section |
 
 ### Table section
 
 ID: `table`
 
-The table section defines the module's
-[indirect function table](AstSemantics.md#calls).
+The encoding of a [Table section](Modules.md#table-section):
 
 | Field | Type | Description |
-| ----- |  ----- | ----- |
-| count | `varuint32` | count of entries to follow |
-| entries | `varuint32*` | repeated indexes into the function section |
+| ----- | ---- | ----------- |
+| element_type | `varuint7` | `0x40`, indicating [`anyfunc`](AstSemantics.md#table) |
+| | `resizable_definition` | see [above](#resizable_definition) |
 
 ### Memory section
 
 ID: `memory`
 
-The memory section declares the size and characteristics of the memory
-associated with the module.
+The encoding of a [Memory section](Modules.md#linear-memory-section) is simply
+a `resizable_definition`:
 
 | Field | Type | Description |
-| ----- |  ----- | ----- |
-| initial | `varuint32` | initial memory size in 64KiB pages |
-| maximum | `varuint32` | maximum memory size in 64KiB pages |
-| exported | `uint8` | `1` if the memory is visible outside the module |
+| ----- | ---- | ----------- |
+| | `resizable_definition` | see [above](#resizable_definition) |
+
+Note that the initial/maximum fields are specified in units of 
+[WebAssembly pages](AstSemantics.md#linear-memory).
+
+### Global section
```

Comment:
It appears that in this patch, the Global section is placed after the Import section. I assume there is a single Global index space; so this means that in the index space, all the imported global indexes will be strictly less than the first locally defined global, is that right? Independently of the final choice (imported global being before or after the locally defined globals), it might be worth telling something about it here or somewhere else (Modules.md), maybe?

Actually, if `call_import` is removed, function calls also share the functions index space and the same precision could be useful too.